apiserver/proxyupdater/proxyupdater_test.go

```diff
@@ -111,17 +111,19 @@ func (s *ProxyUpdaterSuite) TestProxyConfig(c *gc.C) {
 		ProxySettings: params.ProxyConfig{
```

Comment:
does proxyupdater take care to update all the files on disk correctly? The patch looks like we are only setting /etc/juju-proxy.conf from cloud-init, rather than keeping it up-to-date if a user changes anything.